Source/WebKit/UIProcess/Notifications/glib/NotificationService.cpp

```diff
@@ -61,6 +67,48 @@ namespace WebKit {
 
 static const Seconds s_dbusCallTimeout = 20_ms;
 
+#if USE(SKIA)
+
+// Alias to avoid conflicting with write() below.
+ssize_t (*writeToFD)(int, const void*, size_t) = write;
+
+// Simple stream that writes to an fd.
+class fdWStream : public SkWStream {
```

Comment:
The one other thing worth mentioning is that `fdWStream` doesn't follow our usual rules for naming classes (start with capital letter, avoid acronyms). I didn't mention this yesterday because the class is only used in one place, so I don't care very much. And style checker didn't complain. And I understand there is value in keeping the `WStream` suffix as that matches Skia. Still, you could consider `FileDescriptorWStream`.